source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3290,6 +3290,20 @@ class ActionButton {
                     this.action.renderedElement.classList.add(...hostConfig.makeCssClassNames("style-" + this.action.style.toLowerCase()));
                 }
             }
+
+            this.updateAriaExpandedState();
+        }
+    }
+
+    private updateAriaExpandedState() {
```

Comment:
Sorry for being picky, but there are two things I think should be done differently:
- As a general principle, the logic should not rely on inspecting the DOM or checking if a CSS style is applied.
- Instead of introducing yet another method that updates the rendered element's CSS class list, I think we should reuse the existing `updateActionButtonCssStyle`:

  - Change its declaration to add a `buttonState` paremeter of type `ActionButtonState`. To make the method backwards compatible, have that parameter default to `ActionButtonState.Normal`
  - In the `ShowCardAction.updateActionButtonCssStyle` implementation, set `aria-expanded` as appropriate

Reply:
picky is good :)

Working on fix

Reply:
fixed 👍